secant/Mocked Dependencies/KeyStoring.swift

```diff
@@ -34,4 +34,5 @@ protocol KeyStoring {
 enum KeyStoringError: Error {
     case alreadyImported
     case uninitializedWallet
+    case storageError(underlyingError: Error)
```

Comment:
Same question here. Since there is only a single value and it's an error, are we providing any benefit by adding an argument label?

Reply:
good point